cypher-shell/src/main/java/org/neo4j/shell/ShellRunner.java

```diff
@@ -51,7 +53,8 @@ static ShellRunner getShellRunner(@Nonnull CliArgs cliArgs,
             return new InteractiveShellRunner(cypherShell, cypherShell, logger, new ShellStatementParser(),
                     System.in, FileHistorian.getDefaultHistoryFile());
         } else {
-            return new NonInteractiveShellRunner(cliArgs.getFailBehavior(), cypherShell, logger,
+            return new NonInteractiveShellRunner(cliArgs.getFailBehavior(), cypherShell,
+                    new AnsiLogger(cliArgs.getDebugMode(), Format.PLAIN),
```

Comment:
I don't like this. Argument `--format` no longer has any effect for NonInteractive.